sql/hive/src/main/scala/org/apache/spark/sql/hive/client/ClientInterface.scala

```diff
@@ -109,6 +109,9 @@ private[hive] trait ClientInterface {
   /** Returns the name of the active database. */
   def currentDatabase: String
```

Comment:
I would argue that when an interface allows us to set the current database, it should also allow us read it.